<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetRefunds</class>
 <widget class="QWidget" name="WidgetRefunds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remboursements</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExpandAll">
       <property name="text">
        <string>Tout d√©rouler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExport">
       <property name="toolTip">
        <string>Exporter au format CSV</string>
       </property>
       <property name="text">
        <string>Exporter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonChangeDate">
       <property name="text">
        <string>Changer la date</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeViewRefunds"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
